packages/frontend/src/features/sqlRunner/components/CartesianChartStyling.tsx

```diff
@@ -2,42 +2,42 @@ import { XLayoutType } from '@lightdash/common';
 import { Select, Stack, TextInput } from '@mantine/core';
 import debounce from 'lodash/debounce';
 import { Config } from '../../../components/VisualizationConfigs/common/Config';
-import {
-    setXAxisLabel,
-    setYAxisLabel,
-    setYAxisPosition,
-} from '../store/barChartSlice';
+import { type CartesianChartActionsType } from '../store';
 import { useAppDispatch, useAppSelector } from '../store/hooks';
 
 const DEBOUNCE_TIME = 500;
 
-export const BarChartStyling = () => {
+export const LineChartStyling = ({
+    actions,
+}: {
+    actions: CartesianChartActionsType;
+}) => {
     const dispatch = useAppDispatch();
 
     const xAxisLabel = useAppSelector(
         (state) =>
-            state.barChartConfig.config?.display?.xAxis?.label ??
-            state.barChartConfig.config?.fieldConfig?.x?.reference,
+            state.lineChartConfig.config?.display?.xAxis?.label ??
```

Comment:
This one shouldnt be line chart. Will fix

Reply:
Cool! And missing the change on the export const

```
export const CartesianChartStyling = ({
```

Reply:
Fixed